install using cp instead of install for zlib libraries
This patch won't be useful anymore after the upgrade to Buildroot 2009.11.

Signed-off-by: Eric Jarrige <Jorasse@armadeus.org>

Index: buildroot/package/zlib/zlib.mk
===================================================================
--- buildroot/package/zlib/zlib.mk.ori	2008-11-03 00:15:27.000000000 -0800
+++ buildroot/package/zlib/zlib.mk	2009-11-23 16:09:22.908866529 -0800
@@ -40,7 +40,7 @@ $(STAGING_DIR)/usr/lib/libz.so: $(ZLIB_D
 	$(INSTALL) -D $(ZLIB_DIR)/libz.a $(STAGING_DIR)/usr/lib/libz.a
 	$(INSTALL) -D $(ZLIB_DIR)/zlib.h $(STAGING_DIR)/usr/include/zlib.h
 	$(INSTALL) $(ZLIB_DIR)/zconf.h $(STAGING_DIR)/usr/include/
-	$(INSTALL) $(ZLIB_DIR)/libz.so* $(STAGING_DIR)/usr/lib/
+	cp -dpf $(ZLIB_DIR)/libz.so* $(STAGING_DIR)/usr/lib/
 	touch -c $@
 
 $(TARGET_DIR)/usr/lib/libz.so: $(STAGING_DIR)/usr/lib/libz.so
